crates/provider-wit-bindgen/src/lib.rs

```diff
@@ -782,6 +803,8 @@ Please ensure that types for exported interface functions (normally a single rec
         _iface_fn: &wit_parser::Function,
         _cfg: &ProviderBindgenConfig,
     ) -> anyhow::Result<TokenStream> {
+        // TODO: add bundling of exported args to single struct
+        // TODO: handle WIT-map -> Map translation for arguments in iface
```

Comment:
Is this going to be a follow up? If yes, do we need to file an issue before merging?

Reply:
Yup this is a follow up -- just made [an issue](https://github.com/wasmCloud/wasmCloud/issues/999), so we can use that to track it